### DISCUSSION TITLE
uBO 1.44.5b17 + EL Cookie List & Fanboy's Annoyance List Not Updating

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uBlock-issues#2340] [Q&A]
@gorhill,

I have just noticed that my Easylist Cookie List & Fanboy's Annoyance List are not updating (I have not checked other EL filter lists.)

Here is what my filter lists are showing for the last update time.

! Title: Easylist Cookie List
! Updated: 2022-10-21 18:40 UTC

! Title: Fanboy's Annoyance List
! Last modified: 2022-10-20 16:00 UTC

I have tested this on a new Firefox 106.0.1 profile with default settings and only uBO 1.44.5b17 installed. I have tried to force refresh/manually update many times. These lists are not affected by this.

Could this be a regression or side-effect of these recent commits or some other commit?

https://github.com/gorhill/uBlock/commit/2f646dbdb0c297d25a7300804f305df3f87b3cee
https://github.com/gorhill/uBlock/commit/e84b37427901d1ce599e4aeb4ccbdf660880f95a
https://github.com/gorhill/uBlock/commit/83b9ca5db6ab64b8e36082faced8e8f2a74ccb34
https://github.com/gorhill/uBlock/commit/fd945f3358ea371531c5777ff43486094e256061

If not, do you have any other ideas on what could be wrong?

Thank you.

Anyone else having these same issues?
Anyone else have any ideas/solutions as well?

```
uBlock Origin: 1.44.5b17
Firefox: 106
filterset (summary): 
  network: 82133
  cosmetic: 112081
  scriptlet: 22785
  html: 812
listset (total-discarded, last-updated): 
  removed: 
    urlhaus-1: null
    plowe-0: null
  added: 
    fanboy-annoyance: 81514-42084, 16m
    fanboy-cookiemonster: 41908-134, 16m
    ublock-annoyances: 4873-4, 16m
  default: 
    user-filters: 0-0, never
    ublock-filters: 33184-66, 17m
    ublock-badware: 4356-0, 17m
    ublock-privacy: 288-0, 17m
    ublock-abuse: 76-0, 17m
    ublock-unbreak: 1885-0, 17m
    ublock-quick-fixes: 414-0, 17m
    easylist: 62355-314, 16m
    easyprivacy: 30816-1176, 16m
filterset (user): [empty]
modifiedUserSettings: [none]
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 708 ms
  maxAssetCacheWait: 391 ms
```

**Comment by gorhill** [comment]
The commits you list have nothing to do with _Easylist Cookie List_ or  _Fanboy's Annoyance List_.

I think @ryanbr modified the server to cause those lists to be cached virtually forever on Cloudflare edges, probably in response to the situation described in <https://adguard.com/en/blog/easylist-filter-problem-help.html>.

**Reply by ghost** [reply]
Ok, understood.

Thanks for the link to what's going on.

I am not sure if you can answer this or @ryanbr would need to, but when you say those lists are cached virtually forever, does that mean they are never going to receive updates and they are basically frozen static versions of the lists? If that is so, do we need to update the contentURLs to receive updates or is this just status quo until EasyList can figure out a permanent solution or new hosting?

**Reply by gorhill** [reply]
I think it's best to wait for @ryanbr to provide details first about what is going on.

**Reply by ghost** [reply]
Alright, thank you.

I apologize if I sound very ignorant about this whole situation or if I am not using the correct terms or not describing things correctly.

**Comment by gorhill** [comment]
By the way, _Easylist Cookie List_ is part of  _Fanboy's Annoyance List_, you don't need it.

**Reply by ghost** [reply]
Thank you!

**Reply by krystian3w** [reply]
Worse if he like social widgets/links/buttons/newsletters (also in modal these socials and newsletters) and consider disable FA.
